content/en/sensitive_data_scanner/setup/cloud_storage.md

```diff
@@ -11,12 +11,146 @@ further_reading:
+
+When you deploy Agentless scanners using CloudFormation, a single scanner is created per account and scans across all of its regions. You set the region that the scanner is deployed on.
+
+You can add a scanner to a new AWS account or an existing AWS account.
+
+{{< tabs >}}
+{{% tab "New AWS account" %}}
+
+1. Navigate to [Sensitive Data Scanner][1] settings page.
+1. In the **Cloud Settings** section within **Storage**, click **Add AWS accounts by following these steps**.
+1. Leave **Automatically using CloudFormation** enabled.
+1. Select the AWS region in the dropdown menu.
+1. Select an API key that is already configured for Remote Configuration. If the API key you select does not have Remote Configuration enabled, Remote Configuration is automatically enabled for that key upon selection.
+1. If you want to send AWS logs to Datadog, leave **Yes** selected.
+1. Optionally, select **Yes** if you want to use Datadog Cloud Security Management.
+1. **Enable Sensitive Data Scanner** is automatically selected by default. This tells CloudFormation to add the AWS Managed SecurityAudit policy to your Datadog AWS Integration role and enable Agentless Scanning to start scanning your cloud data stores.
+1. Click **Launch CloudFormation Template**.
+
+[1]: https://app.datadoghq.com/organization-settings/sensitive-data-scanner/configuration/data-security
+
+{{% /tab %}}
+{{% tab "Existing AWS account" %}}
+
+1. Navigate to [Sensitive Data Scanner][1] settings page.
+1. In the AWS section within **Storage**:
+    - If you have Agentless scanning already enabled in an account:
+      1. Click the pencil icon for the account.
+      1. Toggle the **Enable Sensitive Data Scanning** to add the scanner to the account.
+      1. Click **Save**.
+    - If you don't have Agentless scanning enabled in an account:
+      1. Click on the plus icon for the account you want to enable sensitive data scanning.
+      1. Select that you want to add the scanner using CloudFormation.
+      1. Select the AWS region in the dropdown menu,
+      1. Select an API key that is already configured for Remote Configuration. If the API key you select does not have Remote Configuration enabled, Remote Configuration is automatically enabled for that key upon selection.
+      1. Toggle the **Enable Sensitive Data Scanning** to add the scanner to the account.
+      1. Click **Launch CloudFormation Template**.
+
+[1]: https://app.datadoghq.com/organization-settings/sensitive-data-scanner/configuration/data-security
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Manually deploy scanners using Terraform {#manually-deploy-scanners-using-terraform}
+
+You can deploy Agentless scanners using the Terraform Module Datadog Agentless Scanner Module. See [Datadog Agentless Scanner Module][7] for more information. Datadog recommends that you choose one of these two setup options if you manually deploy scanners.
```

Comment:
```suggestion
You can deploy Agentless scanners using the [Terraform Module Datadog Agentless Scanner][7]. Datadog recommends that you choose one of these two setup options if you manually deploy scanners:
```

Reply:
I think this is more concise, and doesn't repeat the word "module" as much? But do what is technically right here 🙂